typescript/utils/src/index.ts

```diff
@@ -202,3 +202,5 @@ export { assert } from './validation.js';
 export { BaseValidator, ValidatorConfig } from './validator.js';
 export { tryParseJsonOrYaml } from './yaml.js';
 export { createServiceLogger } from './logging.js';
+export { MINIMUM_GAS, MINIMUM_GAS_ACTION } from './mingas.js';
+export * as AltVM from './altvm.js';
```

Comment:
I'd avoid barrel exports as this might contribute to the bundling issues that we want to solve in the long term

Reply:
Agree, want to keep it in the short term in the AltVM namespace however since this entire file will move into the new interface package that will be created one multi-vm comes along